problems/countNumbers.js

```diff
@@ -12,8 +12,11 @@
  * returns { 99: 2, 11: 1, 12: 1, 13: 1, 58: 1 }
  */
 
-function countNumbers() {
- 
+function countNumbers(arr) {
+    arr = {[]}
```

Comment:
Remember to declare your variables with a let or const in front of them. Also, an empty object requires only curly braces, no square brackets necessary.